src/emmetHelper.ts

```diff
@@ -767,9 +767,15 @@ function getFormatters(syntax: string, preferences: any) {
 			comment: commentFormatter
 		};
 	}
-
+	let fuzzySearchMinScore = preferences.fuzzySearchMinScore || 0.3;
```

Comment:
What if `preferences.fuzzySearchMinScore` is not a number?

Reply:
I have a better one, what if it's 0?